### ISSUE TITLE
Documentation: Inconsistencies in tooltips

### ISSUE THREAD

**Issue by ayshushus** [mozilla/pontoon#4251]
For accessibility, add tooltips to documentation assets, see some examples below for images with missing tooltips:

https://pontoon.mozilla.org/docs/localizer/translation-workspace/#search
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#search-options
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#extra-filters
https://pontoon.mozilla.org/docs/localizer/translate/#performing-batch-actions-on-strings
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#tags
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#context
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#viewing-translation-differences
https://pontoon.mozilla.org/docs/localizer/teams-projects/
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#machinery
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#concordance-search
https://pontoon.mozilla.org/docs/localizer/translation-workspace/#locales
